module/connection/BUILD

```diff
@@ -51,7 +51,7 @@ kt_jvm_library(
         "@maven//:org_slf4j_slf4j_api",
     ],
     resource_strip_prefix = "resources",
-    tags = ["maven_coordinates=com.vaticle.typedb:typedb-studio-dialog:{pom_version}"],
+    tags = ["maven_coordinates=com.vaticle.typedb:typedb-studio-connection-dialog:{pom_version}"],
```

Comment:
Hm, looking through the entire `module` module (heh) it seems like a lot of these tags are using a non-standard value - `connection`, `preference`, `role` and `rule` are all incorrect / out of date.

An example of the standard value is in `module/project/BUILD`:
```py
tags = ["maven_coordinates=com.vaticle.typedb:typedb-studio-module-project:{pom_version}"],
```
The pattern should be: `typedb-studio-{packageNamesSeparatedByHyphens}`.